packages/devextreme/js/ui/card_view.d.ts

```diff
@@ -0,0 +1,1880 @@
+} from '../common/grids';
+import DataSource, { DataSourceLike } from '../data/data_source';
+import Widget, { WidgetOptions } from './widget/ui.widget';
+import { Cancelable, EventInfo, NativeEventInfo } from '../events';
+import { dxToolbarItem, ToolbarItemLocation } from './toolbar';
+import { dxSortableOptions } from './sortable';
+import { dxLoadPanelOptions } from './load_panel';
+import dxScrollable from './scroll_view/ui.scrollable';
+import {
+    Properties as PopupProperties,
+  } from './popup';
+import {
+    Properties as FormProperties, SimpleItem,
+  } from './form';
+import { dxFilterBuilderOptions } from './filter_builder';
+
+/**
+ * @docid
+ * @public
+ * @namespace DevExpress.ui.dxCardView
+ */
+export type Paging = {
+    /**
+     * @docid
+     * @default true
+     * @public
+     */
+    enabled?: boolean;
+    /**
+     * @docid
+     * @default 0
+     * @fires dxCardViewOptions.onOptionChanged
+     * @public
+     */
+    pageIndex?: number;
+    /**
+     * @docid
+     * @default 6
+     * @fires dxCardViewOptions.onOptionChanged
+     * @public
+     */
+    pageSize?: number;
+};
+
+/**
+ * @docid
+ * @public
+ * @namespace DevExpress.ui.dxCardView
+ */
+export type RemoteOperations = {
+    /**
+     * @docid
+     * @default false
+     */
+    filtering?: boolean;
+    /**
+     * @docid
+     * @default false
+     */
+    paging?: boolean;
+    /**
+     * @docid
+     * @default false
+     */
+    sorting?: boolean;
+    /**
+     * @docid
+     * @default false
+     */
+    grouping?: boolean;
+};
+
+// #region Toolbar
+
+/** @public */
+export type PredefinedToolbarItem = 'columnChooserButton' | 'searchPanel' | 'addCardButton' | 'selectAllButton' | 'clearSelectionButton';
+
+/**
+ * @docid
+ * @inherits dxToolbarItem
+ * @public
+ * @namespace DevExpress.ui.dxCardView
+ */
+export type ToolbarItem = dxToolbarItem & {
+    /**
+     * @docid
+     * @public
+     */
+    name?: PredefinedToolbarItem | string;
+    /**
+     * @docid
+     * @default 'after'
+     * @public
+     */
+    location?: ToolbarItemLocation;
+};
+
+/**
+ * @docid
+ * @public
+ * @namespace DevExpress.ui.dxCardView
+ */
+export type Toolbar = {
+    /**
+     * @docid
+     * @public
+     */
+    items?: Array<PredefinedToolbarItem | ToolbarItem>;
+    /**
+     * @docid
+     * @default undefined
+     * @public
+     */
+    visible?: boolean | undefined;
+    /**
+     * @docid
+     * @default false
+     * @public
+     */
+    disabled?: boolean;
+    /**
+     * @docid
+     * @default false
+     * @public
+     */
+    multiline?: boolean;
+};
+
+// #endregion
+
+// #region ColumnsController
+
+/**
+ * @docid
+ * @public
+ * @namespace DevExpress.ui.dxCardView
+ */
+export type FieldInfoType = { // TODO: rename to FieldInfo
```

Comment:
```suggestion
export type FieldInfo = {
```

I see the TODO, just to have a conversation entry for ir

Reply:
Fixed

But @pomahtri warned that the ```FieldInfo``` causes some error
So, if we will face complex issues with this name -> let's rollback my fix to the ```FieldInfoType``` with ```TODO```

Reply:
Hi, @IlyaKhD, @wdevfx. `FieldInfo` causes a conflict with `FieldInfo` from `FilterBuilder`, which is also used in CardView and imported in generated component directly from 'filter_builder' module. This change will lead to re-generation in CardView and **other grids** with `import { FieldInfo } from 'filter_builder' ` replaced to `import { FieldInfo } from 'card_view'` which is totally wrong. So we can revert renaming or use WA with prefixed internal type.